### DISCUSSION TITLE
Does this support mutations?

### DISCUSSION THREAD

**Discussion by kschniedergers** [ottomated/trpc-svelte-query#21] [Q&A] [answered]
Haven't used svelte or trpc before, how do i pass inputs to mutations? I've defined my route like this:

```
	pleasework: publicProcedure
		.input(z.object({ pleeeease: z.string() }))
		.mutation(({ ctx }) => {
			return 'It works!';
		}),
```
and when I try to call the mutation there doesn't seem a way to pass inputs:
```
<script>
	import { trpc } from '$lib/trpc';

	const a = trpc.greeting.query({ name: 'yo' });
    const b = trpc.pleasework.mutation({})
</script>
```

I'm using a fresh create-o7-app with basically just these lines added. the query works, probably something obvious I'm missing :3

**Comment by ottomated** [comment, marked as answer]
`$b.mutate(input)`
